dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx

```diff
@@ -85,30 +82,25 @@ Since the release of version 18.04 (Bionic Beaver) Ubuntu has switched to Netpla
 3. Save the file and exit the text editor.
 4. Set the DNS server information by editing the file `/etc/resolv.conf`:
     ```
-    nano /etc/resolv.conf
+    sudo nano /etc/resolv.conf
     ```
 5. Set the DNS resolver by adding the following lines to the file. Then save it and exit the text editor:
     ```
     nameserver 51.159.47.28
     nameserver 51.159.47.26
     ```
```

Comment:
```suggestion
    ```
    <Message type="tip">
        Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers)
    </Message>
```